actions/xpubs/update.go

```diff
@@ -50,3 +50,45 @@ func (a *Action) update(c *gin.Context) {
 	contract := mappings.MapToXpubContract(xPub)
 	c.JSON(http.StatusOK, contract)
 }
+
+// update will update an existing model
+// Update xPub godoc
+// @Summary		Update xPub
+// @Description	Update xPub
+// @Tags		xPub
+// @Produce		json
+// @Param		Metadata body engine.Metadata false " "
+// @Success		200 {object} models.Xpub "Updated xPub"
+// @Failure		400	"Bad request - Error while parsing Metadata from request body"
+// @Failure 	500	"Internal Server Error - Error while updating xPub"
+// @Router		/v1/users/current [patch]
```

Comment:
we agreed that the endpoints from new API should be with prefix `/api/v1` not just `/v1`

Reply:
changed